src/main/java/com/mycompany/myapp/web/rest/reservation/ReservationResource.java

```diff
@@ -0,0 +1,55 @@
+package com.mycompany.myapp.web.rest.reservation;
+
+import com.codahale.metrics.annotation.Timed;
+import com.mycompany.myapp.domain.Reservation;
+import com.mycompany.myapp.service.reservation.ReservationService;
+import com.mycompany.myapp.web.rest.util.HeaderUtil;
+import io.github.jhipster.web.util.ResponseUtil;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/reservation")
```

Comment:
Can we have an `/api` endpoint and then `/reservation` for all requests with diffrent HTTP methods?

For example HTTP GET for fetching, HTTP delete for deleting and so on... Because having a `/reservation/get` endpoint is silly

So i think about sthing like this:
`@RequestMapping("/api")`

Reply:
i am not 100 proc sure what you want but I'll do my best